### ISSUE TITLE
Update Mutant to be a bit more challenging

### ISSUE THREAD

**Issue by dahaic** [ospaceteam/outerspace#119]
There are multiple low-hanging fruits to employ to get Mutant to be more challenging adversary (not challenging, just more challenging than it is now).

* fleet sizes based on level of given system
  * `Mutant Lodge` less than it is now (12 + 8)
  * `Mutant Den` as it is now (18 + 12)
  * `Mutant Hub` more (24 + 16)
  * `Mutant HQ` having biggest fleets (30 + 20)
  * Two most developed counts, so maximum fleets from big two-gaia systems - 60 + 40
* fleet building based on current inventory, not just pure luck (to prevent excessive fighters to be created)
* possibly able to target all systems in range, with closer ones having just higher chance of attacking (to prevent simple static defenses of today)

In summary, this should, with really small effort, make mutant attack with bigger fleets, more often, and be less predictable. Could be enough for tutorial enemy for now.

**Comment by dahaic** [comment]
All three proposals incorporated. Work has been done throughout PRs #132 and #133
